docs/tooling/cli-cross-chain/teleporter-token-bridge-on-local-networks.md

```diff
@@ -0,0 +1,153 @@
+---
+tags: [Tooling, Avalanche-CLI]
+description: This how-to guide focuses on deploying Teleporter Token Bridge to a local Avalanche network.
+sidebar_label: Teleporter Token Bridge On a Local Network
+pagination_label: Teleporter Token Bridge On a Local Network
+sidebar_position: 3
+---
+# Teleporter Token Bridge
+
+Teleporter Token Bridge enables users to transfer tokens between Subnets. The bridge is a set of 
+smart contracts that are deployed across multiple Subnets, and leverages Teleporter for cross-chain 
+communication.
+
+For more information on Teleporter Token Bridge, check:
+
+- [Teleporter Token Bridge README](https://github.com/ava-labs/teleporter-token-bridge)
+
+# How to Deploy Teleporter Token Bridge on a Local Network
+
+This how-to guide focuses on deploying Teleporter Token Bridge on a local Avalanche network.
+
+After this tutorial, you would have learned how to transfer an ERC-20 token between two 
+Teleporter-enabled Subnets and between C-Chain and a Teleporter-enabled Subnet.
+
+## Prerequisites
+
+For our example, you will first need to create and deploy a Teleporter-enabled Subnet in Local
+Network. We will name our Subnet `testSubnet`.
+
+- To create a Teleporter-enabled Subnet configuration, [visit here](/tooling/cli-cross-chain/teleporter-on-local-networks.md#create-subnet-configurations)
+- To deploy a Teleporter-enabled Subnet, [visit here](/tooling/cli-cross-chain/teleporter-on-local-networks.md#deploy-the-subnets-to-local-network)
+
+## Deploy ERC-20 Token in C-Chain
+
+First, let's create an ERC-20 Token and deploy it to C-Chain. For our example, it will be called 
+TOK. 
+
+Sample script to deploy the ERC-20 Token can be found [here](https://github.com/ava-labs/avalanche-cli/tree/add-deploy-script/deploy_erc20). 
+Note that you will need to modify the script [here](https://github.com/ava-labs/avalanche-cli/blob/add-deploy-script/deploy_erc20/deploy_erc20.sh#L17)
```

Comment:
is that true? why the script needs to be modified? it is assumed to live in the same directory where remappings.txt and lib live